nav2_msgs/action/FollowPath.action

```diff
@@ -22,6 +22,7 @@ std_msgs/Empty result
 uint16 error_code
 string error_msg
 ---
-#feedback definition
-float32 distance_to_goal
-float32 speed
+# feedback definition
+# Real-time tracking error indicating which side of the path the robot is on
+# If the tracking_error is positive, the robot is to the left of the path
+nav2_msgs/TrackingFeedback tracking_feedback
```

Comment:
Where is this feedback used currently in the Nav2 code? This changes so that needs to be adjusted too. I'm thinking the nav2_simple_commander API for `followPath` and the demos that use it perhaps